Framework/Core/src/LogParsingHelpers.cxx

```diff
@@ -25,20 +25,31 @@ char const* const LogParsingHelpers::LOG_LEVELS[(int)LogParsingHelpers::LogLevel
 using LogLevel = o2::framework::LogParsingHelpers::LogLevel;
 
 LogLevel LogParsingHelpers::parseTokenLevel(const std::string &s) {
-  std::smatch match;
-  const static std::regex metricsRE(R"regex(^\[[0-9][0-9]:[0-9][0-9]:[0-9][0-9]\]\[(DEBUG|INFO|STATE|WARN|ERROR)\] .*)regex");
-  std::regex_match(s, match, metricsRE);
 
-  if (match.empty()) {
+  // Example format: [99:99:99][ERROR] (string begins with that, longest is 17 chars)
+  constexpr size_t maxPrefLen = 17;
```

Comment:
According to our coding conventions, constexpr should be capitalised, which I interpret as all capital. @Barthelemy do you confirm?

https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=Variable_Names#Variable_Names

Reply:
Fixed.